### PR TITLE
Changelog and metadata for 0.12.0

### PR DESCRIPTION
**Pull request by HeroicKatora**

### CODE REVIEW

File: Changes.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# v0.12.0
+
+Features:
+- Add compression of pre-compressed frame data, via `Encoder::write_lzw_pre_encoded_frame`.
+- The `color_quant` dependency is now optional. Turning it off disables some
+  interfaces that would internally build quantization tables, which can be
+  costly compared in specialized situations.
```
Comment (fintelia) [inline Changes.md:7]: I'm having trouble parsing this sentence. Did you mean "which can be costly ~compared~ in specialized situations"

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "gif"
 license = "MIT/Apache-2.0"
-version = "0.11.4"
+version = "0.12.0"
 description = "GIF de- and encoder"
 authors = ["nwin <nwin@users.noreply.github.com>"]
```
Comment (fintelia) [inline Cargo.toml:6]:
```suggestion
authors = ["The image-rs Developers"]
```

File: Changes.md (hunk 1 above)
Comment (HeroicKatora) [reply Changes.md]: Agreed, I've rewritten that section.

**Comment by kornelski** [comment]
ping

**Pull request merged**